apps/files_sharing/src/views/SharingDetailsTab.vue

```diff
@@ -245,7 +245,10 @@
 </template>
 
 <script>
+import { emit } from '@nextcloud/event-bus'
+import { fetchNode } from '../services/WebdavClient'
 import { getLanguage } from '@nextcloud/l10n'
+import { logger } from '../services/logger'
```

Comment:
```suggestion
import logger from '../services/logger'
```
Fixes the issue i reported below for me.